app/_src/gateway/kong-enterprise/aws-iam-auth-to-rds-database.md

```diff
@@ -0,0 +1,93 @@
+- Since the Postgres RDS does not support mTLS, you can't enable mTLS between the {{site.base_gateway}} and the Postgres RDS database when AWS IAM authentication is enabled.
+- After enabling AWS IAM authentication in read-write or read-only mode, the `pg_password` or `pg_ro_password` configuration is ignored when establishing connections.
+- You **can't** change the value of the environment variables that you use for the AWS credential after booting {{site.base_gateway}}.
+- A database user assigned to the `rds_iam` role can only authenticate with IAM database authentication.
+
+For additional recommendations and limitations, see [IAM database authentication for MariaDB, MySQL, and PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html) in the Amazon RDS user guide. 
+
+## Prerequisites
+
+Before you enable the AWS IAM authentication, you must configure your AWS RDS database and the AWS IAM role that {{site.base_gateway}} uses.
+
+- **Enable the IAM database authentication on your database instance.** For more information, see [Enabling and disabling IAM database authentication](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.Enabling.html) in the Amazon RDS user guide.
+- **Assign an IAM role to your Kong Gateway instance.** {{site.base_gateway}} can automatically discover and fetch the AWS credentials to use for the IAM role.
+   - If you use an EC2 environment, use the [EC2 IAM role](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html).
+   - If you use an ECS cluster, use a [ECS task IAM role](https://docs.aws.amazon.com/AmazonECS/latest/userguide/task-iam-roles.html).
+   - If you use an EKS cluster, configure a Kubernetes service account that can annotate your assigned role and configure the pods to use an [IAM role defined by serviceaccount](https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html).
+   - If you run {{site.base_gateway}} locally, use the following environment variables: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_PROFILE`, and `AWS_SHARED_CREDENTIALS_FILE`
+   
+   {:.warning}
+   > **Warning:** You **can't** change the value of the environment variables you used to provide the AWS credential after booting {{site.base_gateway}}. Any changes are ignored.
+
+- **Assign an IAM policy to the {{site.base_gateway}} IAM role**. For more information, see [Creating and using an IAM policy for IAM database access](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.IAMPolicy.html) in the Amazon RDS documentation.
+
+- **Ensure you have the database account created in the RDS**. For more information, see [Using IAM authentication with PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.DBAccounts.html#UsingWithRDS.IAMDBAuth.DBAccounts.PostgreSQL) in the Amazon RDS documentation. 
```

Comment:
I saw that you removed those code/terminal snippets, should we keep them to make it convenient for users?(only my idea)

Reply:
@windmgc I removed them because I thought that the linked Amazon doc covered the code snippets (let me know if it's missing something though). For prerequisite sections, we try to keep the instructions shorter otherwise it might belong in the steps instead. And I didn't want to repeat information that Amazon already had published.

Reply:
Okay, sounds good